BuildRobot/module-ci/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN echo 'http://dl-cdn.alpinelinux.org/alpine/edge/community' >> /etc/apk/repos
     python3-dev \
     tiff-dev \
     zlib-dev \
+    linux-headers \
```

Comment:
What error did you run into without this?

Reply:
#27.